From patches Mon Sep 17 00:00:00 2001
From: "Sander Eikelenboom" <linux@eikelenboom.it>
To: qemu-devel@nongnu.org, Sander Eikelenboom <linux@eikelenboom.it>
Cc: Anthony Liguori <aliguori@us.ibm.com>, qemu-devel@nongnu.org,
 kraxel@redhat.com, Sander Eikelenboom <linux@eikelenboom.it>
Date: Mon, 8 Apr 2013 23:08:13 +0200
Message-id: 1365455293-7084-1-git-send-email-linux@eikelenboom.it
Subject: [PATCH 1/1] Update Makefile to actually install the new efi-enabled
 nic roms that are used by default.

Commit c45e5b5b30ac1f5505725a7b36e68cedfce4f01f made a switch use the efi-enabled nic roms by default.
This patch updates the Makefile to install the roms so they can actually be used.

Signed-off-by: Sander Eikelenboom <linux@eikelenboom.it>
Reviewed-by: Gerd Hoffmann <kraxel@redhat.com>
---
 Makefile |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/Makefile b/Makefile
index 80344d9..501daba 100644
--- a/Makefile
+++ b/Makefile
@@ -272,6 +272,8 @@ acpi-dsdt.aml q35-acpi-dsdt.aml \
 ppc_rom.bin openbios-sparc32 openbios-sparc64 openbios-ppc \
 pxe-e1000.rom pxe-eepro100.rom pxe-ne2k_pci.rom \
 pxe-pcnet.rom pxe-rtl8139.rom pxe-virtio.rom \
+efi-e1000.rom efi-eepro100.rom efi-ne2k_pci.rom \
+efi-pcnet.rom efi-rtl8139.rom efi-virtio.rom \
 qemu-icon.bmp \
 bamboo.dtb petalogix-s3adsp1800.dtb petalogix-ml605.dtb \
 multiboot.bin linuxboot.bin kvmvapic.bin \
-- 
1.7.2.5

